packages/opentelemetry-plugin-http/test/utils/SpanAudit.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  SpanKind,
+  Status,
+  Attributes,
+  SpanContext,
+  Link,
+  Event,
+} from '@opentelemetry/types';
+
+export interface SpanAudit {
```

Comment:
Looks like same as `ReadableSpan` interface, Can we use that here? https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-basic-tracer/src/export/ReadableSpan.ts#L26

Reply:
added this issue: https://github.com/open-telemetry/opentelemetry-js/issues/209